client/signup/steps/user/index.jsx

```diff
@@ -270,6 +270,13 @@ export class UserStep extends Component {
 					subHeaderText = translate( 'Already have a WordPress.com account? {{a}}Log in{{/a}}', {
 						components: { a: <a href={ loginUrl } rel="noopener noreferrer" /> },
 					} );
+				} else if ( flowName === HOSTING_LP_FLOW ) {
```

Comment:
There's a helper function that might be more appropriately used here `isHostingSignupFlow`, basically doing the same check as here.